external_auth/oidc-httpd-configs/authentication.conf

```diff
@@ -2,51 +2,67 @@ LoadModule auth_openidc_module modules/mod_auth_openidc.so
 ServerName http://${HTTPD_AUTH_HOST}
 LogLevel   debug
 
-OIDCProviderMetadataURL      https://${HTTPD_AUTH_HOST}:8443/auth/realms/ManageIQ/.well-known/openid-configuration
-OIDCClientID                 ${HTTPD_AUTH_OIDC_CLIENT_ID}
-OIDCClientSecret             ${HTTPD_AUTH_OIDC_CLIENT_SECRET}
-OIDCRedirectURI              http://${HTTPD_AUTH_HOST}/oidc_login/redirect_uri
-OIDCCryptoPassphrase         sp-secret
-OIDCOAuthRemoteUserClaim     username
-OIDCCacheShmEntrySizeMax     65536
 
+OIDCProviderMetadataURL            <%= oidc_provider_metadata_url %>
+OIDCProviderMetadataURL            https://${HTTPD_AUTH_HOST}:8443/auth/realms/ManageIQ/.well-known/openid-configuration
+OIDCClientID                       ${HTTPD_AUTH_OIDC_CLIENT_ID}
+OIDCClientSecret                   ${HTTPD_AUTH_OIDC_CLIENT_SECRET}
+OIDCRedirectURI                    http://${HTTPD_AUTH_HOST}/oidc_login/redirect_uri
+# different from production
+OIDCCryptoPassphrase               sp-secret
```

Comment:
Since this can basically be any string we want...let's just make it match production

Reply:
ok, I'll reencrypt the private key

Reply:
looks like it is the same as pods. leaving it as is